proposals/0005-MicroProfile-1.1.md

```diff
@@ -0,0 +1,61 @@
+# MicroProfile 1.1
+
+* Proposal: [MP-0005](0005-MicroProfile-1.1.md)
```

Comment:
What is the point to use the number? Can we change the process not to contain a number? The number clash occurs quite a few times already. It seems very error-prone.